wagtail/admin/templates/wagtailadmin/generic/form.html

```diff
@@ -7,7 +7,7 @@
         <div class="nice-padding w-mt-8">
             <h2 class="w-relative w-h1" id="header-title">
                 {% icon classname="w-absolute w-top-1 -w-left-11 w-max-w-[1em] w-max-h-[1em]" name=header_icon %}
-                {{ page_subtitle }}
+                {{ page_subtitle|default:page_title }}
```

Comment:
This is also necessary in the case of the accounts view, because we want to show "Account" everywhere (in the `<title>`, the breadcrumbs, and the `<h2>`). Unlike the Create/Edit view, they have separate "title" and "subtitle" for the `<title>`, e.g. "Editing: Muddy Waters".

I tried refactoring the accounts view to be an `EditView` subclass, but it was too different to make it worthwhile.